### PR TITLE
LogForwarding endpoint must be an node name or FQDN, not an IP Address

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1831552

### CODE REVIEW

File: modules/cluster-logging-log-forwarding-about.adoc [hunk 1]
```diff
@@ -13,11 +13,15 @@ If you want to use only the default internal {product-title} Elasticsearch insta
...
 An output can be either:
 
-* `elasticsearch` to forward logs to an external Elasticsearch v5.x cluster, specified by server name or FQDN, and/or the internal {product-title} Elasticsearch instance. 
-* `forward` to forward logs to an external log aggregation solution. This option uses the Fluentd *forward* plug-ins.
+* `elasticsearch` to forward logs to an external Elasticsearch v5.x cluster and/or the internal {product-title} Elasticsearch instance. 
+* `forward` to forward logs to an external log aggregation solution. This option uses the Fluentd *forward* plug-ins. 
+
+[NOTE]
+====
+The endpoint must be a server name or FQDN, not an IP Address.
```
Comment (anpingli) [inline modules/cluster-logging-log-forwarding-about.adoc:23]: @mburke5678  @jcantrill   "Must be"?  The IP Address works in my testing.

Comment (anpingli) [reply modules/cluster-logging-log-forwarding-about.adoc:23]: According to https://bugzilla.redhat.com/show_bug.cgi?id=1825923#c10.  
It should be 'The endpoint must be a server name or FQDN, not an IP Address **when cluster-wide proxy are enabled**.'

Comment (mburke5678) [reply modules/cluster-logging-log-forwarding-about.adoc]: You seem to be right. I was looking at comment 12:
_It's indicating that it should be a server name of FQDN, but I believe that it should be more clear in a Warning indicating that it shouldn't be used an IP Address._

I made the change as you suggested.

**Comment by mburke5678** [comment]
@periklis PTAL

**Review by anpingli — COMMENTED** [review]
LGTM

**Pull request merged**